### PR TITLE
29506 add phone number component

### PR DESCRIPTION
**Pull request by amelieMa**

### CODE REVIEW

File: components/form/Input.spec.js [hunk 1]
```diff
@@ -102,6 +102,24 @@ describe('<Input />', () => {
...
         handleChange={handleChange}
+        onPressEnter={true}
+        t={t}
+      />
+    )
+
+    expect(wrapper.find(Field)).to.have.lengthOf(1)
+  })
+
+  it('should if addons exist', () => {
```
Comment (StanyslasFruchard) [inline components/form/Input.spec.js:113]: Il te manque un verbe ici, comme "It should render if addons exists"

File: components/form/Input.spec.js [hunk 2]
```diff
@@ -124,9 +142,26 @@ describe('<Input />', () => {
 
     expect(wrapper.find(Field)).to.have.lengthOf(1)
   })
+
+  it('should an Input phone', () => {
```
Comment (StanyslasFruchard) [inline components/form/Input.spec.js:146]: Pareil, "it should render an input phone"

File: components/form/Input.spec.js [hunk 3]
```diff
@@ -139,4 +174,10 @@ describe('<InputField /> && <InputFieldPassword />', () => {
 
     expect(wrapInputFieldPassword).to.have.lengthOf(1)
   })
+
+  it('should render InputFieldPhone', () => {
+    const wrapInputFieldPhone = shallow(<InputFieldPhone field={{name: "phone", plcaholder: "your phone"}} />)
```
Comment (kira912) [inline components/form/Input.spec.js:179]: Petite faute de typo plcaholder => placeholder

**Comment by kira912** [comment]
Test OK

**Pull request merged**